libnavigator/src/main/java/com/mapbox/navigation/navigator/SensorMapper.kt

```diff
@@ -0,0 +1,83 @@
+package com.mapbox.navigation.navigator
+
+import android.hardware.Sensor
+import android.hardware.SensorEvent
+import android.os.Build
+import android.os.SystemClock
+import android.util.Log
+import com.mapbox.navigator.Axes3D
+import com.mapbox.navigator.NavigatorSensorData
+import com.mapbox.navigator.SensorData
+import com.mapbox.navigator.SensorType
+import java.time.Instant
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+object SensorMapper {
```

Comment:
Because this class lives in the SDK, it deserves some tests. Especially for the time estimates